data-reconciliation-app/docs/data-schema-data-flow.md

```diff
@@ -1,18 +1,34 @@
 # Investigate Data Structure and Sample data
 
-Goal: Investigate data reconciliation sample app data flow, how we will reconcile this data in our application and potential sample data sets. We met with Brent to discuss potential use of LEI data as a test case for our sample application. While LEI may be a test case for our application, we will build the app to accept any data schema to extend to many reconciliation scenarios.
+Goal: Investigate data reconciliation sample app data flow, how we will reconcile this data in our application and potential sample data sets. We met with Brent to discuss potential use of reference data as a test case for our sample application. While reference data may be a test case for our application, we will build the app to accept any data schema to extend to many reconciliation scenarios.
 
 ## Generic Data Reconcilition App = Many Use Cases
 
-We will build our application for use across any data schemas. Pending legal approvals, LEI may be a test case for our app.
+We will build our application for use across any data schemas. Pending legal approvals, reference data may be a test case for our app.
 
 Main idea: We build a generalized sample app for a bunch of institutions who want to collaborate on some set of data, and share results out on that data.
 
-## Data Schema
+## Data Input Schema
 
-Assumption: In our sample application, members will agree upon their schema. The only requirement is that this schema has a unique idenifier and attributes associated with the unique identifier.
+Requirements:
 
-Our main audience for our sample app is a developer. A developer could take our sample app and extend it to their own use case. A developer could provide their input schema. Therefore, validation on the schema is not neccessary for our app, but it could be a simple check:
+- Members will agree upon their schema in advance - the only requirement is that this schema has a unique idenifier and attributes associated with the unique identifier.
+- MVP: Our idenitifier and attributes types will be strings.
+- MVP: We will only handle one attribute in this data reconciliation app. For example:
+
+  | unique_id | attribute_1 |
+  | --------- | ----------- |
+  | A001      | XX          |
+  | A002      | XY          |
+  | A003      | XZ          |
+
+- TODO: Brent to provide initial sample data set
+- Stretch: Attribute types can be numerical as well.
+- Stretch: Multiple attributes per unique id
+
+## Data Schema Validation
+
+Our main audience for our sample app is a developer. A developer could take our sample app and extend it to their own use case. A developer could provide their input schema. Validation on the schema is not neccessary for our app, but it could be a simple check:
```

Comment:
Nit:
```suggestion
Our main audience for our sample app is developers. A developer could take our sample app and extend it to their own use case. A developer could provide their input schema. Validation on the schema is not necessary for our app, but it could be a simple check:
```